build/templates/tox-system_tests.ini.mako

```diff
@@ -57,19 +61,36 @@ commands =
     # token is from codecov
     ${module_name}-coverage: codecov -X gcov --token=4c58f03d-b74c-489a-889a-ab0a77b7809f --no-color --flags ${module_name}systemtests --name ${module_name} --root ../.. --file ../../generated/${module_name}/coverage.xml
 
+    ${module_name}-test_suite: python --version
+    ${module_name}-test_suite: python -c "import platform; print(platform.architecture())"
+    ${module_name}-test_suite: python -c "import ${module_name}; ${module_name}.print_diagnostic_information()"
+    ${module_name}-test_suite: python -m pytest ../../src/${module_name}/system_tests --json=generated/kibana/${module_name}_system_test_result.json {posargs}
```

Comment:
In #1268 I added `--durations`.
This may need integration whichever goes in second.

Reply:
Noted